src/util/kaldi-table-inl.h

```diff
@@ -340,6 +410,12 @@ template<class Holder>  class SequentialTableReaderScriptImpl:
     // or the user called FreeCurrent().. note,
     // if when called by user we are in this state,
     // it means the user called FreeCurrent().
+    kObjectLoaded, // Called NextScpLine() and found the next    yes         yes
+    // object we need was alreay loaded in the last line.
```

Comment:
Did you mean LoadCurrent()? there is no LoadObject() here. Yep "last" is not accurately. I think in my implementation kObjectLoaded just describes: we already have an object in the holder, corresponding to data_rxfilename_ in the *next* line. It doesn't matter that the range was specified or not (in the current line or next line)